# this variable declares different states of the robot
uint8 state

uint8 OFF = 0
uint8 CONFIGURABLE = 1
uint8 IDLE = 1
uint8 ERROR = 2
uint8 CONNECTING = 3
uint8 OP_ANGLES = 4
uint8 OP_TORQUES = 5